test/Microsoft.TemplateEngine.TestTemplates/Microsoft.TemplateEngine.TestTemplates.csproj

```diff
@@ -11,10 +11,14 @@
         <NoWarn>$(NoWarn);NU5128</NoWarn>
         <IsPackable>true</IsPackable>
         <IsShipping>false</IsShipping>
+        <NoDefaultExcludes>true</NoDefaultExcludes>
     </PropertyGroup>
     <ItemGroup>
         <Compile Remove="**\*"  />
         <Content Include="test_templates\**\*" Exclude="test_templates\**\bin\**;test_templates\**\obj\**" />
         <None Include="test_templates\**\*" CopyToOutputDirectory="Always" />
+
+        <Content Include="nupkg_templates\**" />
```

Comment:
I'm not fan of storing nupkgs in the repo? Can `PackageManager` be used instead? (see test.utils)

Reply:
I agree, 1 nuget package was in repo before, I did add another one, we should probably switch to using PackageManager, but I would prefer to do this at later time.